### ISSUE TITLE
What's this?

### ISSUE THREAD

**Issue by JamesWClark** [Jacob1233/changeHash#2]
What's all this, Jacob? Start school again or doing a job? What's up with you?

**Comment by Jacob1233** [comment]
Hey! I graduated K-State Cyber Security Boot Camp last fall but I don't have any tech job or anything. I had some interviews but I'm currently working as campus security at Avila University. This project is a precursor to another project where I am learning where I can inject magic bytes into images. I'm slowly getting back in to programming and hoping that some of my projects might catch a potential employers attention. How have you been?

**Comment by JamesWClark** [comment]
Yeah, my advice is to just keep doing projects, but really you should be finishing projects most importantly. Once you've got a few projects together you can create a portfolio from it and employers will want to see that. It's great ammo for your interviews. As for me, still teaching at the rock. Computer science is stronger than ever with some prerequisite courses creating more depth and college credit through UMKC.
